project-base/app/translations/dataFixtures.cs.po

```diff
@@ -1078,6 +1081,9 @@ msgstr "JURA Impressa J9 TFT Carbon"
 msgid "Job at Shopsys"
 msgstr "Práce v Shopsys"
 
+msgid "Join our team and be part of an innovative company. Explore exciting career opportunities and grow with us!"
+msgstr "Přidejte se k nám a buďte součástí inovativní společnosti. Objevte vzrušující kariérní příležitosti a rostejte s námi!"
```

Comment:
it is just demo data, however, "rostejte" sounds quite weird, is it a real word? :smile:

Reply:
https://cs.wiktionary.org/wiki/r%C5%AFst#%C4%8Dasov%C3%A1n%C3%AD :smile: